src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml

```diff
@@ -32,6 +32,7 @@
 			Visibility="{x:Bind EmptyRecentsTextVisibility, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
 		<ListView
 			x:Name="RecentsView"
+			Margin="-14,-8,-12,0"
```

Comment:
Negative margins aren't usually recommended, what's the reason for using them here?

Reply:
I'll fix by overriding Expander container style.